pages/linux/tcpick.md

```diff
@@ -0,0 +1,17 @@
+# tcpick
+
+> It is a command-line tool in Linux used for packet sniffing and network traffic analysis.
+> It allows you to capture and display TCP connections and their data. You can use it to monitor network traffic on a specific interface or capture traffic to and from a particular host or port.
+> More information: <https://linux.die.net/man/8/tcpick>.
+
+- Basic Usage:
+
+`tcpick -i {interface} -C -h {{host}} -p {{port}}`
+
+- Advanced Options:
+
+`man tcpick`
```

Comment:
We normally don't use the man page command as an example. You can replace it with something like `tcpick --help` and change the description "Advanced Option" to "Show help".

Reply:
@blueskyson Changes are done.